src/OpenSage.Game/Data/Map/BuildListInfo.cs

```diff
@@ -0,0 +1,224 @@
+﻿using System.IO;
+using System.Numerics;
+using OpenSage.Data.Ini;
+using OpenSage.FileFormats;
+
+namespace OpenSage.Data.Map
+{
+    // TODO: Does this belong under Data.Map anymore?
```

Comment:
Yeah, let's move this to `OpenSage.Logic.Map` (but doesn't have to be in this PR). In general (ha), let's follow the C++ folder structure unless there's an actual reason to differ.